test/micro/org/openjdk/bench/jdk/preview/patterns/Exactness.java

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
The exactness predicates should have regression tests on their correct operation in addition to any performance testing.

Reply:
I have this regarding numeric values

test/langtools/tools/javac/patterns/PrimitiveInstanceOfNumericValueTests.java

Reply:
> I have this regarding numeric values. Is it the kind of test you have in mind?
> 
> [test/langtools/tools/javac/patterns/PrimitiveInstanceOfNumericValueTests.java](https://github.com/openjdk/jdk/pull/15638/files#diff-0c7b37309d920b4c1af93789aed337c9eb259ad31977b9ef395e7b8c4b47a6d6)

IMO, the library methods merit some of their own dedicated direct testing (since an end user could in principle call them too) in addition to indirect tests of their operation via javac's code generation.

Reply:
I did a translation of that file into [ExactnessConversionsSupportTest.java](https://github.com/openjdk/jdk/pull/15638/commits/4e5ac772417a2e50d47fd6fa8feb363a6bffa88c#diff-c24740028b9956d2dd2947907713f5e2d8681e8762f6d8e26e4faf2ded3979a8). What else could I add?